Makefile.am

```diff
@@ -17,6 +17,9 @@ usbguard_notifier_CXXFLAGS=\
 	@notify_CFLAGS@ \
 	@usbguard_CFLAGS@
 
+appicondir=/usr/share/usbguard-notifier/icons
```

Comment:
This path should be modifiable.

Reply:
Does it work? The name of the folder might be **usbguard_notifier.**

Reply:
it does work but I am changing this to make it configurable by the user